node/src/reactor/joiner.rs

```diff
@@ -518,8 +518,8 @@ impl reactor::Reactor for Reactor {
                                 block_header_with_old_version,
                             }) => {
                                 let old_version = block_header_with_old_version.protocol_version();
-                                error!(%current_version, %old_version, "downgrade is not allowed, exiting");
-                                panic!("downgrade is not allowed");
+                                fatal!(effect_builder, "downgrade is not allowed, exiting; current_version={}; old_version={}", current_version, old_version).await;
```

Comment:
If rustfmt doesn't want to, let's fix the formatting manually.

Reply:
I think "downgrade is not allowed" is hard to understand for the user.
Maybe something like: "network is still running an older version"?

Reply:
* rustfmt did format the code when I temporarily moved it out of the closure :)
* message has been changed

Both fixed in https://github.com/casper-network/casper-node/pull/2434/commits/246344895b1ea995ae917d35ffdeef1c1b084e30